build_tools/azure/install.sh

```diff
@@ -66,10 +77,25 @@ python_environment_install_and_activate() {
         source $VIRTUALENV/bin/activate
         pip install -r "${LOCK_FILE}"
 
-    elif [[ "$DISTRIB" == "pip-nogil" ]]; then
-        python -m venv $VIRTUALENV
+    elif [[ "$DISTRIB" == "pip-free-threaded" ]]; then
+        python3.13t -m venv $VIRTUALENV
         source $VIRTUALENV/bin/activate
         pip install -r "${LOCK_FILE}"
+        # TODO for now need pip 24.1b1 to find free-threaded wheels
+        pip install -U --pre pip
+        # TODO When there are CPython 3.13 free-threaded wheels for numpy and
+        # scipy move this to
+        # build_tools/azure/cpython_free_threaded_requirements.txt. For now we
+        # install them from scientific-python-nightly-wheels
+        dev_anaconda_url=https://pypi.anaconda.org/scientific-python-nightly-wheels/simple
+        dev_packages="numpy scipy"
+        pip install --pre --upgrade --timeout=60 --extra-index $dev_anaconda_url $dev_packages
+        # TODO Move cython to
+        # build_tools/azure/cpython_free_threaded_requirements.txt when there
+        # is a CPython 3.13 free-threaded wheel
+        # For now, we need the development version of Cython which has CPython
+        # 3.13 free-threaded fixes so we install it from source
```

Comment:
About 30hours ago a cython wheel appeared on https://anaconda.org/scientific-python-nightly-wheels/Cython/files (I think someone worked on this during the scientific python summit?)

Is it worth using that instead of building from source?

Reply:
Nice, I am a bit confused though that there is only a pure Python wheel and not python version specific wheels like there is on PyPI ...

Reply:
is it because cython itself doesn't contain any compiled code/c extensions?